### ISSUE TITLE
FP S1192 don't raise duplications on JSX classnames

### ISSUE THREAD

**Issue by saberduck** [SonarSource/SonarTS#574]
React classnames are used as identifiers, so they  cannot and shouldn't be refactored using a constant.

“bubble-popup-section” is marked as duplicated

```typescript
return (
  <BubblePopup customClass="source-viewer-bubble-popup" position={popupPosition}>
    <div className="bubble-popup-section">{line.scmAuthor}</div>
    {line.scmDate && (
      <div className="bubble-popup-section">
        <DateFormatter date={line.scmDate} />
      </div>
    )}
    {line.scmRevision && <div className="bubble-popup-section">{line.scmRevision}</div>}
  </BubblePopup>
);
```

https://jira.sonarsource.com/browse/RSPEC-1192
